src/app/src/main/kotlin/org/vaccineimpact/api/app/Helpers.kt

```diff
@@ -63,4 +66,30 @@ fun Future<Exception?>.awaitAndThrowIfError()
     {
         throw exception
     }
+}
+
+@Suppress("UNCHECKED_CAST")
```

Comment:
These are here because I found DataTable.new was raising a 'no primary constructor' error if i didn't cast data explicitly now there are a choice of types of wide and long rows. Not sure if there's a better approach..